cmd/vsphere-xcopy-volume-populator/README.md

```diff
@@ -198,15 +215,6 @@ from the ConfigMap under the 'powermax' namespace, which the CSI driver uses.
   standard error reasons and vanish after the next try. If the popoulator fails the migration
   can be restarted. We may want to restart/retry that populator or restart the migration.
 
-- VIB issues
```

Comment:
vib issues should stay, as we are keeping the vib

Reply:
yes